prov/sm2/src/sm2.h

```diff
@@ -328,29 +334,15 @@ int sm2_cntr_open(struct fid_domain *domain, struct fi_cntr_attr *attr,
 
 int64_t sm2_verify_peer(struct sm2_ep *ep, fi_addr_t fi_addr);
 
-void sm2_format_pend_resp(struct sm2_tx_entry *pend, struct sm2_cmd *cmd, void *context,
-			  enum fi_hmem_iface iface, uint64_t device,
-			  const struct iovec *iov, uint32_t iov_count, uint64_t op_flags,
-			  int64_t id, struct sm2_resp *resp);
-void sm2_generic_format(struct sm2_cmd *cmd, int64_t peer_id, uint32_t op, uint64_t tag,
-			uint64_t data, uint64_t op_flags);
-size_t sm2_copy_to_sar(struct smr_freestack *sar_pool, struct sm2_resp *resp,
-		       struct sm2_cmd *cmd, enum fi_hmem_iface, uint64_t device,
-		       const struct iovec *iov, size_t count, size_t *bytes_done,
-		       int *next);
-size_t sm2_copy_from_sar(struct smr_freestack *sar_pool, struct sm2_resp *resp,
-			 struct sm2_cmd *cmd, enum fi_hmem_iface iface, uint64_t device,
-			 const struct iovec *iov, size_t count, size_t *bytes_done,
```

Comment:
There's a lot in this patch (like the above) that should be part of the removal patch, not this one.

Reply:
The reason these were removed in the "Change to Nemesis patch" is they no longer compile after the change to Nemesis.  I think moving the removal patch to before the Nemesis patch would help.  I can't remember why I put it after, but I was pretty sure I did it for a reason... time to re-discover it :).